src/Features/CSharp/Portable/Completion/SuggestionMode/CSharpSuggestionModeCompletionProvider.cs

```diff
@@ -91,6 +93,20 @@ private bool IsAnonymousObjectCreation(SyntaxToken token)
             return false;
         }
 
+        private bool IsPossibleTupleExpression(SyntaxToken token)
```

Comment:
Just verified that behavior did not change for
```C#
            System.Console.WriteLine(a|
            System.Console.WriteLine(a|)
```
You do need a double paren to get this behavior in an arg list

```C#
            System.Console.WriteLine((a|
```

I think this is not too bad for a prototype